### DISCUSSION TITLE
an addon/extention for alert feed

### DISCUSSION THREAD

**Discussion by Sharkacuga** [Chatterino/chatterino2#4931] [Ideas]
You'll have a list of alerts from stream elements, sound alerts, Blerp Sound ect. in a list of who did the alert. i did some concepts to get an idea
![Screenshot 2023-11-02 000001](https://github.com/Chatterino/chatterino2/assets/57742974/9eaa05cb-63be-46d6-805b-9f6c6ab31164)
![Screenshot 2023-11-02 000002](https://github.com/Chatterino/chatterino2/assets/57742974/c99d8e0c-53ca-41b7-93ee-e8a28a38cca0)

**Comment by Mm2PL** [comment]
I'd love for plugins to be able to do something like this: custom channels and fully custom message building.
